Gemfile

```diff
@@ -24,10 +24,6 @@ group :development, :test, :docker_development do
   gem 'puma'
   gem 'listen'
   gem 'rspec-rails'
-
-  # Used by Mailatcher
-  gem 'sinatra', github: 'sinatra'
-  gem 'mailcatcher'
```

Comment:
`mailcatcher` should not be included in the `Gemfile`. See: https://github.com/sj26/mailcatcher#bundler